app/controllers/container_node_controller.rb

```diff
@@ -7,6 +7,10 @@ class ContainerNodeController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
+  def show_list
+    process_show_list(:where_clause => 'container_nodes.deleted_on IS NULL')
```

Comment:
right, both currently don't work. I thought to wait for a solution and then continue with this PR, so we don't merge something that we know is not working. Anyway, I agree they all should be consistent so I'll make the change.